tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -155,42 +142,26 @@ def test_snapshot_create_latency(
     microvm_factory,
     guest_kernel,
     rootfs,
-    firecracker_release,
     st_core,
 ):
-    """
-    Test scenario: Full snapshot create performance measurement.
-
-    Testing matrix:
-    - Guest kernel: all supported ones
-    - Rootfs: Ubuntu 18.04
-    - Microvm: 2vCPU with 512 MB RAM
-    """
+    """Measure the latency of creating a Full snapshot"""
 
-    # The guest kernel does not "participate" in snapshot restore, so just pick some
-    # arbitrary one
-    if "4.14" not in guest_kernel.name():
-        pytest.skip()
-
-    logger = logging.getLogger("snapshot_sequence")
+    # The guest kernel does not "participate" in snapshot restore, so just pick
+    # some arbitrary one
+    if "4.14" not in guest_kernel.name:
+        pytest.skip("just test one guest kernel")
```

Comment:
now that we have the guest_kernel_4_14 fixture, can we get rid of these to eliminate some `[SKIPPED]`s?

Reply:
I spent a bit looking at this, but it did not work because the tests use `st_core`, which itself uses `guest_kernel`. We can work around it by doing:

```python
only_one_guest_kernel = pytest.mark.parametrize(
    "guest_kernel", list(kernel_params("vmlinux-4.14*")), indirect=True
)

@only_one_guest_kernel
def test_snapshot_create_latency(
...
```

I will add it as a separate commit.